clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -26,10 +21,6 @@ spec:
       # Start with 2Gi and add 2Gi for each 2K nodes.
       memory: {{MultiplyInt 2 (AddInt 1 (DivideInt .Nodes 2000))}}Gi
       {{end}}
-  ruleSelector:
```

Comment:
`ruleSelector` is not only used for alerting, but also for recording rules. We rely on recording rules to compute Network Programming SLO, hence the failed presubmit. Please restore this section. I'd suggest maybe to remove `role: alert-rules` label and replace it with something more meaningful (here and in prometheus-rule.yaml file).

Reply:
Thanks, good catch. Done.